src/sage/rings/polynomial/pbori/pbori.pxd

```diff
@@ -1,14 +1,13 @@
 from libcpp.memory cimport unique_ptr, shared_ptr, make_shared
 
-from sage.rings.polynomial.multi_polynomial_ring_base cimport \
-                                                MPolynomialRing_base
+from sage.rings.polynomial.multi_polynomial_ring_base cimport MPolynomialRing_base, BooleanPolynomialRing_base
```

Comment:
Is it still necessary to cimport  `MPolynomialRing_base`? It is the only instance in this file. Do any file importing from `sage.rings.polynomial.pbori.pbori` expect it? If so, could they import it directly from `sage.rings.polynomial.multi_polynomial_ring_base` instead?

Reply:
It is used in `pbori.pyx` for an `isinstance` check

Reply:
I can of course move the `cimport` to the `pyx` file if you think that's better

Reply:
I do not think it is particularly better. A cython expert would have to chime in on that. So, let's leave it as is.